bedrock/firefox/templates/firefox/hello/index-2016-b.html

```diff
@@ -0,0 +1,22 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/hello/index-2016.html" %}
```

Comment:
I think we're OK with this, but I'll confirm with Jen. Otherwise I can do the same thing for the current page as well, we'll just want to consider how redesigning the page in the middle of the test would impact data.